extensions/functions_boolean.yaml

```diff
@@ -0,0 +1,35 @@
+%YAML 1.2
+---
+scalar_functions:
+  -
+    name: not
+    description: The boolean not of a provided value
+    impls:
+      - args:
+          - value: boolean
+          - value: boolean
+        return: boolean
+  -
+    name: or
+    description: The boolean or of two values.
+    impls:
+      - args:
+          - value: boolean
+          - value: boolean
+        return: boolean
+  -
+    name: and
```

Comment:
Should this have any discussion of TVL/Kleene? Or are you thinking that's a separate set of functions?

Reply:
I was thinking this was three-valued. We should clarify in the descriptions. Were you thinking it shouldn't be?

(in general, I think in sql)

Reply:
TVL sounds right to me.